services/director-v2/tests/unit/test_modules_dynamic_sidecar_scheduler_task.py

```diff
@@ -0,0 +1,143 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+
+import asyncio
+import re
+from typing import Final, Iterator
+
+import httpx
+import pytest
+import respx
+from _pytest.monkeypatch import MonkeyPatch
+from fastapi import FastAPI
+from pytest_simcore.helpers.typing_env import EnvVarsDict
+from respx.router import MockRouter
+from simcore_service_director_v2.models.schemas.dynamic_services.scheduler import (
+    SchedulerData,
+)
+from simcore_service_director_v2.modules.dynamic_sidecar.scheduler.events import (
+    REGISTERED_EVENTS,
+    DynamicSchedulerEvent,
+)
+from simcore_service_director_v2.modules.dynamic_sidecar.scheduler.task import (
+    DynamicSidecarsScheduler,
+)
+
+SCHEDULER_INTERVAL_SECONDS: Final[float] = 0.1
+
+# FIXTURES
+
+
+@pytest.fixture
+def mock_env(
+    mock_env: EnvVarsDict,
+    monkeypatch: MonkeyPatch,
+    simcore_services_network_name: str,
+    docker_swarm: None,
+    mock_docker_api: None,
+) -> None:
+    monkeypatch.setenv(
+        "DIRECTOR_V2_DYNAMIC_SCHEDULER_INTERVAL_SECONDS",
+        f"{SCHEDULER_INTERVAL_SECONDS}",
+    )
+
+    monkeypatch.setenv("DIRECTOR_V2_DYNAMIC_SCHEDULER_ENABLED", "true")
+
+    monkeypatch.setenv("S3_ENDPOINT", "")
+    monkeypatch.setenv("S3_ACCESS_KEY", "")
+    monkeypatch.setenv("S3_SECRET_KEY", "")
+    monkeypatch.setenv("S3_BUCKET_NAME", "")
+
+    monkeypatch.setenv("POSTGRES_HOST", "")
```

Comment:
why do you set this to ""?? That is an strange setup ...

Reply:
Because they are not required for the test. The intent is to suggest they are irrelevant for the test.

Reply:
that is not what it suggested to me :-/

Reply:
I've changed it slightly now and it should be more meaningful. let me know if it makes more sense now